hledger/hledger.cabal

```diff
@@ -193,7 +193,7 @@ executable hledger
     , safe >=0.2
     , shakespeare >=2.0.2.2
     , split >=0.1
-    , statistics
+    , statistics <0.15
```

Comment:
I'd merge this commit with the previous "Minor infrastructure fixes", and rename to "set statistics upper bound; update gitignore" (or split those)